<launch>
<node pkg="automatic_cart" type="backend.py" name="CartBackend" output="screen">
</node>
</launch>
